src/app/shared/palette/order-summary/read-order-summary-field.component.spec.ts

```diff
@@ -14,7 +14,7 @@ describe('ReadOrderSummaryFieldComponent', () => {
 
   const FIELD_TYPE: FieldType = {
     id: 'PersonOrderSummary',
-    type: 'OrderSummary'
```

Comment:
This is fixing old bug where OrderSummary was considered a new base type. It is a predefined complex type.